tests/sentry/api/endpoints/test_organization_metrics.py

```diff
@@ -288,6 +289,7 @@ def test_span_measurement_samples(self):
 
         for i, mri in enumerate(
             [
+                "g:spans/mobile.slow_frames@none",
```

Comment:
I'm not sure if it's meaningful for me to update the tests further considering the other MRIs would use the same kind of mechanism for this test